src/notification/NotificationStatus.tsx

```diff
@@ -0,0 +1,49 @@
+import axios from 'axios';
+import { useToken } from '../contexts/TokenProvider';
+import { IToken } from '../types/token';
+import { useNotificationStatus } from '../contexts/NotificationStatusProvider';
+import { INotification, INotificationStatus } from '../types/notification';
+import { useState, useEffect } from 'react';
+
+const NotificationStatus = () => {
+  const [notificationStatusList, setNotificationStatusList] = useState<boolean[]>([]);
+  const tokenContextObj: IToken | null = useToken();
+  const notificationStatusContextObj: INotificationStatus | null = useNotificationStatus();
+
+  const fetchNotificationData = async () => {
+    await axios
+      .get('http://kdt.frontend.3rd.programmers.co.kr:5006/notifications', {
+        headers: { Authorization: `bearer ${tokenContextObj?.token}` },
+      })
+      .then((res) => {
+        const serverData = res.data;
+        const allSeenCheckData = serverData.map((data: INotification) => data.seen);
+        setNotificationStatusList(allSeenCheckData);
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+
+  const checkNotificationStatus = () => {
+    if (notificationStatusList.indexOf(false) !== -1) notificationStatusContextObj?.setNotification(true);
+  };
+
```

Comment:
```suggestion
const checkNotificationsStatus = () => {
    if(!notificationStatusList.includes(false)) notificationStatusContextObj?.setNotification(true);
    };
```
로 수정해도 코드가 더 깔끔하게 보일 수 있을 것 같습니다☺️